configs/config.development.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
How will current config be called then? Just `config.json`?

Anyway, naming things IMO is completely up to a particular project.